andela/ah-jawans-backend
# #166790085 Send email verification on successful signup

#### What does this PR do?
          Send email on successful registration to verify the user
#### Description of Task to be completed?
          When a user creates account he should receive an email on successful registration in order to verify his account
#### How should this be manually tested?
          -open Postman
          - type the URL (localhost:3000/api/users)
          - input username, email and password, and click send button
          - Go to your email and click on the link
          - copy the link opened in the URL field and paste it in Postman
          - hit send button to verify email.
#### Any background context you want to provide?
           For a user to create an account he has to verify to make sure he is using his email.
#### What are the relevant pivotal tracker stories?
         #166790085 An Email should be sent to a user upon successful registration. User should be able to receive a verification link via email upon successful signu### Questions:
